internal/terraform/exec/exec_test.go

```diff
@@ -13,7 +14,7 @@ func TestExec_timeout(t *testing.T) {
 		SleepDuration: 100 * time.Millisecond,
 		Stdout:        "Terraform v0.12.0\n",
 	})
-	e.SetWorkdir("/tmp")
+	e.SetWorkdir(os.TempDir())
```

Comment:
Does this create any files? May be cleaner to create a tmp dir and remove it on a t.Cleanup? https://golang.org/pkg/io/ioutil/#TempDir

Reply:
It doesn't - or at `version` execution in particular shouldn't and there is no other code that would. It just essentially needs to `cd` into a valid directory on the filesystem, so it should be safe to just use just `os.TempDir()` here, I think, but I'm working through the other tests to assess the situation there

Reply:
I have pulled this out into a separate PR: https://github.com/hashicorp/terraform-ls/pull/73
Review is welcomed there. 😄